frontend/packages/console-shared/src/utils/resource-utils.ts

```diff
@@ -556,10 +557,23 @@ export class TransformResourceData {
     );
   };
 
+  getActiveStatefulSets = (ss: K8sResourceKind): K8sResourceKind[] => {
+    const { statefulSets } = this.resources;
+    const ownedRS = _.filter(statefulSets?.data, (f) => f.metadata.name === ss.metadata.name);
+    return _.filter(ownedRS, (rs) => _.get(rs, 'status.replicas'));
+  };
+
   public getReplicaSetsForResource = (deployment: K8sResourceKind): PodControllerOverviewItem[] => {
     const replicaSets = this.getActiveReplicaSets(deployment);
     return sortReplicaSetsByRevision(replicaSets).map((rs) =>
-      getIdledStatus(this.toReplicaSetItem(rs), deployment),
+      getIdledStatus(this.toResourceItem(rs, ReplicaSetModel), deployment),
+    );
+  };
+
+  public getStatefulSetsResource = (ss: K8sResourceKind): PodControllerOverviewItem[] => {
```

Comment:
The intention is for this to really be `getReplicaSetForStatefulSet`, correct?  In that it parallels `getReplicaSetForResource`, but `StatefulSet` is not a `Deployment`?